app/models/concerns/document_delivery_requestable.rb

```diff
@@ -61,7 +61,18 @@ def one_step_doc_delivery?(phys_holdings, user = nil)
     check_for_one_step
   end
 
+  def two_step_doc_delivery?(phys_holdings, user = nil)
+    links = doc_delivery_links(phys_holdings, user)
+    check_for_two_step = links.count == 1 && links.first[:urls].size > 1
+    @two_step_links = links.first[:urls] if check_for_two_step
```

Comment:
@alexBLR Shouldn't this return the whole list of links? What if the record had multiple holdings that were available?